io.github.CatalystDevOrg.Catalyst/io.github.CatalystDevOrg.Catalyst.json

```diff
@@ -0,0 +1,64 @@
+{
+  "app-id": "io.github.CatalystDevOrg.Catalyst",
+  "base": "org.electronjs.Electron2.BaseApp",
+  "base-version": "23.08",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "run.sh",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=x11",
+    "--socket=pulseaudio",
+    "--device=dri",
+    "--talk-name=org.freedesktop.Notifications",
+    "--talk-name=org.kde.StatusNotifierWatcher",
+    "--talk-name=org.gtk.Notifications",
+    "--own-name=io.github.CatalystDevOrg.Catalyst"
+  ],
+  "modules": [
+    {
+      "name": "catalyst",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x catalyst_3.5.4_amd64.deb",
```

Comment:
so hw does this work on aarch64?

Reply:
Catalyst can be built for various additional architectures (like ARM). The official build system however doesn't output these, but community builds (though outdated) are available.

Reply:
so how does it work with aarch64?

Reply:
or you could build from source here. Like it should.

Reply:
Building from source would be preferable.

Reply:
Source builds fail with this error: https://github.com/flatpak/flatpak-builder-tools/issues/381

Reply:
What command did you use to build it and what's the entire command output

Reply:
@jdev082 It happened with this tutorial when generating source list: https://docs.flatpak.org/en/latest/electron.html#bundling-npm-packages

Reply:
Fixed with single-arch flathub.json